apps/frontend/src/components/global/checklist/ChecklistSeverityOverride.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <v-card class="text-center px-8 pt-2" height="300px">
+    <v-card-title class="justify-center"
+      >Severity Override Justification</v-card-title
+    >
+    <v-card-subtitle
+      v-if="selectedRule.severityJustification === ''"
+      class="justify-center mt-1"
+    >
+      <strong>
+        Please input a valid severity override justification. (Required)
+      </strong>
+    </v-card-subtitle>
+    <v-card-subtitle v-else class="justify-center mt-1">
+      <strong>Press "OK" to save.</strong>
+    </v-card-subtitle>
+    <v-textarea
+      v-model="selectedRule.severityJustification"
+      class="mt-2"
+      solo
+      outlined
+      dense
+      no-resize
```

Comment:
worthwhile to explore auto-grow instead of no-resize.  also remove explicit height statement (or at least just make it a min height sorta deal)

Reply:
Now using `auto-grow` and height is removed